### PR TITLE
1st negative number in every window size of k

### PR DESCRIPTION
**Pull request by bitaashna**
#2304
#3265 
Description

One of the variable sliding window problem has been solved and implemented using c++.

Checklist

I've read the contribution guidelines.
I've checked the issue list before deciding what to submit.
I've edited the README.md and link to my code.
#C++
#2304

### CODE REVIEW

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -236,6 +236,12 @@ _add list here_
 - [Minimum number of insertions and deletions](dp/min_ins_del.cpp)
 - [Unbounded Knapsack](dp/unbounded_knapsack.cpp)
 
+## Sliding Window
+
+
```
Comment (siddharth25pandey) [inline C-Plus-Plus/README.md]: Remove the blank lines

File: C-Plus-Plus/README.md [hunk 2]
```diff
@@ -236,6 +236,12 @@ _add list here_
...
+- [1st negative number in every window size of k](sliding_window/1st_negative_no_in_every_window_of_size_k.cpp)
+
+
```
Comment (siddharth25pandey) [inline C-Plus-Plus/README.md]: Remove the blank lines

File: C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	take two pointers say i,j 
+	Initialise i from 0 and  j from i to k and find the 1st negative.
+	In this , we are doing repetitive work 
+	therefore it's time complexity O(n^2)
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+int main(){
+	int n;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp:17]: Please use a good naming convention

File: C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+			    //slide the window
+				i++;
+				j++;
+
+		}
+
+	}
+	for(int i=0;i<ans.size();i++)
+		cout<<ans[i]<<" ";
```
Comment (siddharth25pandey) [inline C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp:66]: Display a message (before this) what you are printing for better user-interface.

File: C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+	Initialise i from 0 and  j from i to k and find the 1st negative.
+	In this , we are doing repetitive work 
+	therefore it's time complexity O(n^2)
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+int main(){
+	int n;
+	cin>>n;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp:19]: Display message what you are taking input from the user

File: C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+	int arr[n];
+	for(int i=0;i<n;i++)
+		cin>>arr[i];
```
Comment (siddharth25pandey) [inline C-Plus-Plus/sliding_window/1st_negative_no_in_every_window_of_size_k.cpp:24]: Display message what you are taking input from the user

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@bitaashna Do the suggested changes

**Comment by siddharth25pandey** [comment]
@bitaashna Also, no need to create multiple commits for a single PR

**Comment by bitaashna** [comment]
> @bitaashna Do the suggested changes

I have made the changes @siddharth25pandey 
Please check it.

**Comment by siddharth25pandey** [comment]
@bitaashna Too many commits for single PR. Better to create a new PR

**Pull request closed without merging**